Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Loading/LoadingCode.bind

```diff
@@ -0,0 +1,72 @@
+                           Stretch="UniformToFill"
+                           HorizontalAlignment="Center"
+                           VerticalAlignment="Center" />
+                </Grid>
+            </DataTemplate>
+        </ResourceDictionary>
+    </Page.Resources>
+
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+        <ScrollViewer>
+            <StackPanel Margin="20">
+                <RichTextBlock Margin="0,10,0,0" Foreground="{StaticResource Brush-Grey-03}">
+                    <Paragraph FontSize="18" FontWeight="Bold">The loading control is for showing an animation with some content when the user should wait in some tasks of the app.</Paragraph>
+                    <Paragraph />
+                    <Paragraph>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat. Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi.</Paragraph>
+                    <Paragraph />
+                    <Paragraph>Expetenda tincidunt in sed, ex partem placerat sea, porro commodo ex eam. His putant aeterno interesset at. Usu ea mundi tincidunt, omnium virtute aliquando ius ex. Ea aperiri sententiae duo. Usu nullam dolorum quaestio ei, sit vidit facilisis ea. Per ne impedit iracundia neglegentur. Consetetur neglegentur eum ut, vis animal legimus inimicus id.</Paragraph>
+                    <Paragraph />
+                    <Paragraph>His audiam deserunt in, eum ubique voluptatibus te. In reque dicta usu. Ne rebum dissentiet eam, vim omnis deseruisse id. Ullum deleniti vituperata at quo, insolens complectitur te eos, ea pri dico munere propriae. Vel ferri facilis ut, qui paulo ridens praesent ad. Possim alterum qui cu. Accusamus consulatu ius te, cu decore soleat appareat usu.</Paragraph>
+                </RichTextBlock>
+
+                <controls:AdaptiveGridView
+                Margin="0,12,0,0"
+                x:Name="AdaptiveGridViewControl"
+                ItemHeight="200"
+                DesiredWidth="300"
+                SelectionMode="None"
+                IsItemClickEnabled="False"
+                ItemTemplate="{StaticResource PhotosTemplate}" />
+
+            </StackPanel>
+        </ScrollViewer>
+
+        <controls:Loading x:Name="LoadingControl" LoadingHorizontalAlignment="@[LoadingContentHorizontalAlignment:Enum:HorizontalAlignment.Center:Left.Right.Center.Stretch]" 
+                          LoadingVerticalAlignment="@[LoadingContentVerticalAlignment:Enum:VerticalAlignment.Center:Stretch.Center.Top.Bottom]"
```

Comment:
I believe LoadingVerticalAlignment and the like should be changed to the "base" properties.
Example:
LoadingVerticalAlignment -- VerticalContentAlignment